### PR TITLE
[webui] Use package_name instead of package

### PR DESCRIPTION
**Pull request by DavidKang**
In case of multibuild, we always stripped the multibuild suffix.

### CODE REVIEW

**Comment by DavidKang** [comment]
@bgeuken, I will do it

File: src/api/app/models/package.rb [hunk 1]
```diff
@@ -928,10 +928,13 @@ def buildresult(prj = project, show_all = false)
     LocalBuildResult::ForPackage.new(package: self, project: prj, show_all: show_all)
   end
 
-  def jobhistory_list(project, repository, arch, limit = 100)
-    results = Jobhistory.find_hashed(project: project.name, package: name,
+  def jobhistory_list(project, repository, arch, options = {})
+    options[:limit] = 100 if options[:limit].blank?
```
Comment (Ana06) [inline src/api/app/models/package.rb]: `options[:limit] |= 100` :wink:

File: src/api/app/models/package.rb [hunk 2]
```diff
@@ -928,10 +928,13 @@ def buildresult(prj = project, show_all = false)
...
+    options[:package] = name if options[:package].blank?
```
Comment (Ana06) [inline src/api/app/models/package.rb]: And I would say here you can also use `options[:package] |= name`

File: src/api/app/views/webui/packages/job_history/index.html.haml [hunk 3]
```diff
@@ -31,7 +31,7 @@
           = time_tag(Time.at(jobhistory.ready_time))
         %td
           - if jobhistory.reason == 'source change'
-            = link_to(jobhistory.reason, package_rdiff_path(project: @project.name, package: @package.name, orev: jobhistory.prev_srcmd5, rev: jobhistory.srcmd5))
+            = link_to(jobhistory.reason, package_rdiff_path(project: @project.name, package: @package, orev: jobhistory.prev_srcmd5, rev: jobhistory.srcmd5))
```
Comment (Ana06) [inline src/api/app/views/webui/packages/job_history/index.html.haml]: it is not really related with the changes.. better in a different commit :thinking:

**Comment by Ana06** [comment]
It seems I arrived to late and this is already merged... :sob:

**Pull request merged**